op-batcher/batcher/txmgr.go

```diff
@@ -28,28 +32,64 @@ type TransactionManager struct {
 	// Outside world
 	txMgr    txmgr.TxManager
 	l1Client *ethclient.Client
+	relayer  *bitcoinda.Relayer
 	signerFn opcrypto.SignerFn
 	log      log.Logger
 }
 
 func NewTransactionManager(log log.Logger, txMgrConfg txmgr.Config, batchInboxAddress common.Address, chainID *big.Int, senderAddress common.Address, l1Client *ethclient.Client) *TransactionManager {
+	relayer, err := bitcoinda.NewRelayer(bitcoinda.Config{
+		Host:         "op-bitcoin:18332",
+		User:         "rpcuser",
+		Pass:         "rpcpass",
+		HTTPPostMode: true,
+		DisableTLS:   true,
+	})
+	if err != nil {
+		fmt.Println(err)
+		return nil
```

Comment:
Is not returning an error here intentional? If so, maybe an inline comment to explain why ignore the error would be nice.

Reply:
It should be returning the error. Nice catch!